app/Dao/UserDao.php

```diff
@@ -108,4 +99,33 @@ public function deleteUserById($id): void
         $user = User::findOrFail($id);
         $user->delete();
     }
+
+    /**
+     * Login user by email and password
+     * @param Request data
+     * @return User
+     */
+    public function checkLogin($user): ?object
+    {
+        $email = $user['email'];
+        $foundUser = User::where('email', $email)->first();
+        if (!$foundUser) {
+            return null;
+        }
+        return $foundUser;
```

Comment:
Need to check this. May be $found user is null when email doesn't match. 
You don't need to check to return null or $foundUser. Just return that.